generators/chipyard/src/main/scala/Clocks.scala

```diff
@@ -47,6 +47,12 @@ case object ClockingSchemeKey extends Field[ChipTop => Unit](ClockingSchemeGener
   */
 case object ClockFrequencyAssignersKey extends Field[Seq[(String) => Option[Double]]](Seq.empty)
 case object DefaultClockFrequencyKey extends Field[Double]()
+case object ReferenceClockTrackerKey extends Field[ReferenceClockTracker](new ReferenceClockTracker)
+class ReferenceClockTracker {
```

Comment:
Using a `Field` to pass values between Module and Harness feels wrong.
Perhaps just set harnessFreq as a property of ChipTop, and then in Harness, do
```
val harnessFreq = lazyDut match { 
  case d: ChipTop => d.harnessFreq
  case _ => // some default
}
```

Reply:
Done